stun_server/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "stun_server_ip-us_east_1" {
```

Comment:
Lets do 1 output for "ip" that is a dictionary containing all of them.

Reply:
Created an output `stun_server_ip` with all IPs.

Reply:
They are all “stun_server” that is what this workspace is

Reply:
Fine. 👍🏻